src/components/shared/NewsletterPopup.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useEffect, useState} from "react";
+import {Button, Icon} from "semantic-ui-react";
+
+export default function NewsletterPopup() {
+    const [hidden, setHidden] = useState(true)
+
+    useEffect(() => {
+        if (!localStorage.getItem("SubscribePopupSeen")){
+            if(localStorage.getItem("returnUser") === "true") {
+                setTimeout(function () {
+                    setHidden(false)
+                }, 30000) // 30 seconds in ms
+            }
+        }
+    }, [])
+
+    const onClickHandle = () => {
+        setHidden(true)
+        localStorage.setItem("SubscribePopupSeen", "true")
+    }
+
+    return (
+        <div className={"newsletter-popup"} style={{visibility: hidden ? "hidden" : "visible"}}>
+            <Button className={"newsletter-popup-close-button"} icon onClick={onClickHandle}>
```

Comment:
Also can they only close this popup if they click a button? Is there any way we can get them out of the popup by clicking on the area outside the popup (which is more of a conventional UI pattern from my experience)